### DISCUSSION TITLE
Github rejected my request for Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by tjamesbu** [community/community#29369] [Actions]
I entered my information that I am currently a grad student at University of Michigan and a photo of my UMich student ID.  My permanent address is still from the USA, but I am currently down in Quito, Ecuador studying my Masters program through UMich online.  UMich treats online Masters students as normal students and presents them with the same student ID on-campus students receive.  I believe Github has detected my current location (Ecuador) and found it does not match with the school information I entered, and that is the reason it rejected me.  Is there someone I can email who can fix this, or do I need to just reattempt to apply and not allow permission for Github to detect my current location?  Any suggestions/solutions you can offer would be helpful.

Thanks!

**Comment by samuel-ma** [comment]
Hi  there @tjamesbu 

I believe there's nothing wrong with the documents you provided Github, it takes 2-3days for Github to verify your identity and give the student pack

Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

If you've provided any of the following requirements, it will take Github 2-3 days to verify your identity

**Reply by tjamesbu** [reply]
I was rejected within 10mins after sending the request.  I provided a photo of my student Id from University of Michigan that has an expiration month/year on it to prove its credibility.  I received a message from github stating my location didn’t correlate with the school name I selected.  GitHub also stated that if I am a student in a virtual program, the student Id has to have the label ‘virtual student’ on it.  UMich does not consider its online students as ‘virtual students’ since we are paying the same tuition prices as on-campus students…and therefore does not provide student IDs with the virtual label github is requesting.

**Reply by samuel-ma** [reply]
This sounds like an issue for Github stuff then
Contact them [here](https://support.github.com/)
If you fail to find a solution, come back here, I'll be happy to find an alternative way to help
